File: py/Program/Vehicle.py
class Vehicle:
    __plat = ""
    __merk = ""
    __tahunProduksi = 0
    __warna = ""
    __pemilik = ""

    def __init__(self, plat = "", merk = "", tahunProduksi = 0, warna = "", pemilik=""):
        self.__plat = plat
        self.__merk = merk
        self.__tahunProduksi = tahunProduksi
        self.__warna = warna
        self.__pemilik = pemilik

    def setPlat(self, plat):
        self.__plat = plat
    
    def getPlat(self):
        return self.__plat
    
    def setMerk(self, merk):
        self.__merk = merk
    
    def getMerk(self):
        return self.__merk

    def setTahunProduksi(self, tahunProduksi):
        self.__tahunProduksi = tahunProduksi
    
    def getTahunProduksi(self):
        return self.__tahunProduksi

    def setWarna(self, warna):
        self.__warna = warna
    
    def getWarna(self):
        return self.__warna
    
    def setPemilik(self, pemilik):
        self.__pemilik = pemilik

    def getPemilik(self):
        return self.__pemilik

File: cpp/Program/Vehicle.cpp
#pragma once
#include <bits/stdc++.h>

using namespace std;

class Vehicle{
    private:
        string plat;
        string merk;
        int tahunProduksi;
        string warna;
        string pemilik;
    
    public:
        Vehicle(){
          
        }

        Vehicle(string plat, string merk, int tahunProduksi, string warna, string pemilik){
            this->plat = plat;
            this->merk = merk;
            this->tahunProduksi = tahunProduksi;
            this->warna = warna;
            this->pemilik = pemilik;
        }

        void setPlat(string plat){
            this->plat = plat;
        }

        string getPlat(){
            return this->plat;
        }

        void setMerk(string merk){
            this->merk = merk;
        }

        string getMerk(){
            return this->merk;
        }

        void setTahunProduksi(int tahunProduksi){
            this->tahunProduksi = tahunProduksi;
        }

        int getTahunProduksi(){
            return this->tahunProduksi;
        }

        void setWarna(string warna){
            this->warna = warna;
        }

        string getWarna(){
            return this->warna;
        }

        void setPemilik(string pemilik){
            this->pemilik = pemilik;
        }

        string getPemilik(){
            return this->pemilik;
        }

        virtual int getJmlKursi(){
            return -1;
        }

        virtual int getJmlPintu(){
            return -1;
        }

        virtual string getJenis(){
            return "";
        }

        virtual int getTangki(){
            return -1;
        }

        ~Vehicle(){

        }
};